app/src/lib/app-state.ts

```diff
@@ -148,7 +148,7 @@ export type Popup = { type: PopupType.RenameBranch, repository: Repository, bran
                     { type: PopupType.Preferences } |
                     { type: PopupType.MergeBranch, repository: Repository } |
                     { type: PopupType.RepositorySettings, repository: Repository } |
-                    { type: PopupType.AddRepository } |
+                    { type: PopupType.AddRepository, initialPath: string | null } |
```

Comment:
Looks like we both did the same thing https://github.com/desktop/desktop/blob/871354c931e897138324e0f03deab9a89e49f201/app/src/lib/app-state.ts#L151